test/contrib/test_socks.py

```diff
@@ -199,6 +206,8 @@ class TestSocks5Proxy(IPV4SocketDummyServerTestCase):
     Test the SOCKS proxy in SOCKS5 mode.
     """
     def test_basic_request(self):
+        disable_warnings(NoIPv6Warning)
```

Comment:
Hmm. Neither `conftest.py`, nor `__init__.py`, nor sticking it at the top-level of the test file do the trick. I could put it in the `setUp()` call? Thoughts, @Lukasa?